app.js

```diff
@@ -64,36 +64,33 @@ catchphraseButton.addEventListener('click', () => {
     // get the value of the catchphrase input
 
     const newCatch = catchphraseInput.value;
-    console.log(newCatch);
     // push the new catchphrase to the catchphrase array in state
     catchphrases.push(newCatch);
     // clear out the form input's value so it's empty to the user
     catchphraseInput.value = '';
     displayCatchphrases();
-    console.log(catchphrases);
     // update the dom to show the new catchphrases (refactor to/call displayCatchphrases to do this work)
 
 });
  
 function displayStats() {
     // text content of the reportEl to tell the user how many times they've changed each piece of the state
-    timesChanged.textContent = `you changed the head ${changeHead} times, the middle ${changeMiddle} times, the bottom ${changeBottom} times.`;
+    reportEl.textContent = `you changed the head ${changeHead} times, the middle ${changeMiddle} times, the bottom ${changeBottom} times.`;
```

Comment:
This function is causing an error because `reportEl` is `null` and does not have a `.textContent` property